erdpy-up.py

```diff
@@ -278,6 +275,6 @@ def __init__(self, message, inner=None):
 
     logger.info("""
 
-For more information go to https://docs.elrond.com.
+For more information go to https://docs.multiversx.com.
```

Comment:
The Telegram group has to be renamed as well (but not now, I think).

Reply:
I don't know if it's been renamed.